Package.swift

```diff
@@ -0,0 +1,33 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "TreeSitterPython",
+    products: [
+        .library(name: "TreeSitterPython", targets: ["TreeSitterPython"]),
+    ],
+    dependencies: [],
+    targets: [
+        .target(name: "TreeSitterPython",
+                path: ".",
+                exclude: [
+                    "binding.gyp",
+                    "bindings",
+                    "Cargo.toml",
+                    "corpus",
+                    "grammar.js",
+                    "LICENSE",
```

Comment:
hmm, maybe don't exclude the license?

Reply:
This exclude list only controls what the Swift system builds. Everything except for sources must be excluded. However, because this PR was sitting for so long, more recent versions of SPM are now better-behaved in the face of non-compilable sources. So, I actually don't think anything needs to be excluded in this case for it to function right.